Can't find GeneratedKey.snk in a new enlistment

Repro:

1. Fork this repo and clone it.
2. From the root directory, run `BuildAndTest.cmd`.

Expected: It builds.

Actual:

`CSC : error CS7027: Error signing output with public key from file 'GeneratedKey.snk' -- File not found. [e:\Repos\sarif-sdk\src\Sarif\Sarif.csproj]`

Cause: The script creates `GeneratedKey.snk` in the root directory, but the project files don't refer to it at the correct path.

Fix: Modify the project files to refer to the key at the correct path. (The project file Sarif.Viewer.VisualStudio.csproj has it right. Sarif.csproj and Sarif.Driver.csproj have it wrong.)

--- a/src/Sarif.Driver/Sarif.Driver.csproj
+++ b/src/Sarif.Driver/Sarif.Driver.csproj
@@ -19,7 +19,7 @@
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
   <PropertyGroup>
-    <AssemblyOriginatorKeyFile>GeneratedKey.snk</AssemblyOriginatorKeyFile>
+    <AssemblyOriginatorKeyFile>..\..\GeneratedKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
@@ -149,7 +149,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="app.config" />
-    <None Include="GeneratedKey.snk" />
+    <None Include="..\..\GeneratedKey.snk" />
     <None Include="packages.config">
       <SubType>Designer</SubType>
     </None>

--- a/src/Sarif/Sarif.csproj
+++ b/src/Sarif/Sarif.csproj
@@ -12,7 +12,7 @@
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
   <PropertyGroup>
-    <AssemblyOriginatorKeyFile>GeneratedKey.snk</AssemblyOriginatorKeyFile>
+    <AssemblyOriginatorKeyFile>..\..\GeneratedKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
   <ItemGroup>
@@ -90,7 +90,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="app.config" />
-    <None Include="GeneratedKey.snk" />
+    <None Include="..\..GeneratedKey.snk" />
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>

